tests/mutations.rs

```diff
@@ -997,6 +977,53 @@ fn confirm_history() {
     assert_eq!(event.tick, tick);
 }
 
+#[test]
+fn after_disconnect() {
```

Comment:
I added a test to ensure that after disconnect mutation acks are dropped before processing.